packages/block-library/src/columns/index.js

```diff
@@ -22,6 +22,7 @@ export const settings = {
 	icon,
 	variations,
 	example: {
+		viewportWidth: 600, // Columns collapse "@media (max-width: 599px)".
```

Comment:
Note that this is an existing API. The `viewportWidth` property already exists. Previously we didn't need to set if for these blocks, because the preview viewport would be the same as the main parent window. A lot of previews are setting the default viewport width to 500. Maybe we should change the default to be a bit larger.